### PR TITLE
Feature/home page cr

### PR DESCRIPTION
**Pull request by Fordo29**
### 1. What changed?
Updated the Home page view and created the corresponding components to show all the pets.  

### 2. Why is this change necessary?
This is necessary for the initial set of of the architecture of the Home page view

### 3. What changed technically that may impact others?
Nothing changed technically other than showing components of where items are displayed

### 4. How do we test it?
`npm start` and see there is a Home page started.

### CODE REVIEW

File: src/Components/PetContainer.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import '../Styles/PetContainer.scss';
+import PetCard from './PetCard';
+
+
+const PetContainer = () => {
```
Comment (StevieAmb) [inline src/Components/PetContainer.js:6]: This is the component where we will need to bring in data that we iterate through to populate the pet card.

File: src/Styles/Home.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+@import '_variables.scss';
```
Comment (anewb87) [inline src/Styles/Home.scss:1]: Getting sassy really quick with the variables. Really love how efficient this is.

File: src/Components/PetContainer.js (hunk 1 above)
Comment (Fordo29) [reply src/Components/PetContainer.js:6]: Yes maybe we can create some dummy data to iterate through

File: src/Components/PetCard.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import '../Styles/PetCard.scss';
+
+
+const PetCard = () => {
```
Comment (StevieAmb) [inline src/Components/PetCard.js:5]: I very much appreciate you building this out and being explicit about what's going to show on the pet card.

File: src/Styles/Home.scss (hunk 2 above)
Comment (Fordo29) [reply src/Styles/Home.scss:1]: This is definitely a smart thing to keep things dynamic and easier down the road.

File: src/Components/PetCard.js (hunk 3 above)
Comment (Fordo29) [reply src/Components/PetCard.js:5]: This is just generic to get something to show on the page but I appreciate the kudos

File: src/index.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { BrowserRouter } from 'react-router-dom';
 import ReactDOM from 'react-dom';
-import './Styles/index.css';
+import './Styles/base.scss';
```
Comment (anewb87) [inline src/index.js:4]: Bringing in the reset file is key NOW as we move forward, so we don't have to retroactively 'fix' css things.

**Review by anewb87 — APPROVED** [review]
Like the basic outline of the Home page.

**Pull request merged**